cities/hamburg.json

```diff
@@ -1160,7 +1160,8 @@
             "properties" : {
                 "opening_hours" : "Sa 08:00-16:00",
                 "title" : "Flohschanze",
-                "location" : "Neuer Kamp 30"
+                "location" : "Neuer Kamp 30",
+                "details_URL" : "https://www.hamburg.de/flohmarkt/4245446/flohschanze/"
```

Comment:
Please change `details_URL` to `details_url` to have all attributes consistent in lowercase.

Reply:
I'll make this change

Reply:
Done